build/media_source/system/js/keepalive.es6.js

```diff
@@ -26,4 +26,4 @@ if (keepAliveUri === '') {
   keepAliveUri = `${(systemPaths ? `${systemPaths.root}/index.php` : window.location.pathname)}?option=com_ajax&format=json`;
 }
 
-setInterval(() => navigator.sendBeacon(keepAliveUri), keepAliveInterval);
+setInterval(() => fetch(keepAliveUri, {method: 'POST'}), keepAliveInterval);
```

Comment:
```suggestion
setInterval(() => fetch(keepAliveUri, { method: 'POST' }), keepAliveInterval);
```
This should fix javascript cs in drone.